source/documentation/team-information/internal-processes/assess-reserved-instances.html

```diff
@@ -0,0 +1,85 @@
+
+There are a number of steps to gather data and evaluate RIs to ensure that we are purchasing the best opportunities and minimise risk of waste.
+
+### Gather recommendations
+
+Use AWS [Reserved Instances Recommendations](https://docs.aws.amazon.com/cost-management/latest/userguide/ri-recommendations.html) via the Billing and Cost Management Console, to indentify potential RI optimisation opportunities.
+
+Depending on the type of resources there are some filters that can be applied to identify appropriate recommendations:
+
+- **Offering class** - This is EC2 specific and allows you to select **"[Convertible](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ri-convertible-exchange.html)"** RIs. This can help mitigate risk of waste by converting the RI to a different type.
+- **Recommendation level** - Select **"Payer"**. This allows us to purchase and use RIs across all accounts within the MoJ Organisation.
+- **Term** - Select **"1-year"**. We do not purchase RIs over a year to mitigate risk of waste. We can consider this is the future as our FinOps practice matures.
+- **Payment option** - Select **"No upfront"**. This provides saving rates based on no upfront payment. This offers the lowest saving, but due to Commercial constraints we are not able to make upfront commitments for resources.
+- **Based on the last** -  Select one of 7, 30 or 60 days. This is the look back period for analysis of usage.
+
+This will generate a set of recommendations that can be assessed. There are a number of factors we will look at that will determine whether we will accept a recommendation.
+
+### Assessment against defined criteria
+
+The strongest recommendations will meet a larger number if these criteria:
+
+1. Recommedation aligns to [Reserving Infrastucture Capacity](https://cloud-optimisation-and-accountability.justice.gov.uk/documentation/finops-and-greenops-at-moj/patterns/reserved-instances.html#reserving-infrastructure-capacity) guidance?
+
+2. Resource is not already covered by our AWS Compute Savings Plan i.e. also purchasing an RI would lead to over commitment, and therefore waste.
+
+3. Usage trends have been consistent for the last 6 to 12 months
+
+> NB: We may still consider resources with less usage history but it will be critical to understand future usage via engamenet with the resource owners. See Engagement
+
+4. Resources are for "production" services i.e. less likley to see signifcant change that may impact usage
+
+5. Estimated usage of RI close to 100% i.e. no waste
+
+### Engagement
+
+The based on the data and our assessment criteria an RI recommedation may appear a good commitment. However, without engagement with the owner of those resources we may purchase a year long commitment for a resources that may be tuned off a week later. This is waste as we would be committed to pay for a resource that is not being used. This is why it is essential to discuss plans for the usage of these resources with the owners. 
```

Comment:
'The based on the data and our assessment criteria an RI recommedation may appear a good commitment'

Should this be 'When based...'  Also typo in 'recommendation'

Reply:
Word salad. Structure and typos fixed.